docs/topics/vscode-extension-resolving-issues.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * docs/vs-code-extension-guide/master.adoc
+
+[id='vscode-extension-resolving-issues_{context}']
+
+= Resolving issues with the {ProductName} extension
+
+You can resolve issues by using a Quick Fix or by editing the code yourself.
+
+.Procedure
+
+. Resolve issues detected by the {ProductName} extension by performing one of the following actions:
+
+* If the issue displays the Quick Fix icon (image:vs_optional.png[Optional or Quick Fix]), you can right-click the issue and select *Apply All QuickFixes*.
+
+* If the issue has any other icon or you decide not to use all suggested Quick Fixes, right-click the issue, select *Open Code*, make any changes needed to the code, and then save the file.
+
+. Right-click the resolved issue and select either *Mark as Resolved* or *Delete*.
```

Comment:
Are you sure about "resolved issue"? Why would the user want to mark a resolved issue as resolved or to delete a resolved issue?

I am pretty sure that "Mark as Resolved" is not used for issues that the user has fixed. I think it is for issues that the user does not want to change. That was what I saw when I looked at the UI.

I would make this into a bullet: * Right-click the issue and select "Mark as Resolved". 

I don't think you need to document "Delete". It is obvious.